pythonscript/bindings/builtins_binder/vector2.cpp

```diff
@@ -262,6 +262,19 @@ static mp_obj_t _make_new_vector2(const mp_obj_type_t *type, size_t n_args, size
 }
 
 
+static mp_obj_t _unary_op_vector2(mp_uint_t op, mp_obj_t in) {
+  auto self = static_cast<Vector2Binder::mp_godot_bind_t*>(MP_OBJ_TO_PTR(in));
+  switch (op) {
+    case MP_UNARY_OP_NEGATIVE:
```

Comment:
I like my solution better, just check it. Why? Because I'm not creating new objects :P